packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1193,6 +1192,11 @@ export default class SalesforceCRMService implements CRM {
             writeOnRecordBody[field.name] = picklistValue;
             continue;
           }
+        } else if (field.type === SalesforceFieldType.CHECKBOX) {
+          // If the checkbox field value is not a boolean for some reason, default to if it's a falsely value
+          const checkboxValue = !!fieldConfig.value;
```

Comment:
Should be a boolean value but if someone inputs 1 or "true" text we should evaluate as truthy.